openedx/core/djangoapps/catalog/tests/factories.py

```diff
@@ -107,6 +107,14 @@ class SeatFactory(DictFactoryBase):
     upgrade_deadline = factory.LazyFunction(generate_zulu_datetime)
 
 
+class EntitlementFactory(DictFactoryBase):
+    currency = 'USD'
+    price = factory.Faker('random_int')
+    sku = factory.LazyFunction(generate_seat_sku)
+    mode = 'verified'
```

Comment:
Nit: Is there a constant defined anywhere in edx-platform for this that we could use here? If not, would it make sense to create one so that we can start weaning ourselves off of using the strings directly in logic?